Holo-annie-jam/Managers/ScreenManager/GameScreen.cs

```diff
@@ -32,6 +32,11 @@ public enum ScreenState {
 /// as screens.
 /// </summary>
 public abstract class GameScreen {
+    #region  Fields
+
+    private bool _isLoaded;
```

Comment:
iirc underscore precedes variable name only when it's private readonly? nitpicking tho it doesn't actually matter that much